machine_test.go

```diff
@@ -1645,3 +1649,28 @@ func TestCreateSnapshot(t *testing.T) {
 		})
 	}
 }
+
+func testCreateBalloon(ctx context.Context, t *testing.T, m *Machine) {
+	if err := m.CreateBalloon(ctx, 6, true); err != nil {
+		t.Errorf("Create balloon device failed from testAttachBalloon: %s", err)
+	}
+}
+
+func testGetBalloonConfig(ctx context.Context, t *testing.T, m *Machine) {
+	expectedBalloonConfig := models.Balloon{
+		AmountMb:     Int64(6),
+		DeflateOnOom: Bool(true),
+	}
```

Comment:
It may be better to define 6 as a constant. If one of them is updated, another must be updated as well.

Reply:
Do you think it be better if I wrap the balloon info into Machine.Config struct?